app/views/admin/edition_tags/edit.html.erb

```diff
@@ -51,11 +51,21 @@
       </div>
       <div class="publishing-controls well">
         <div class="form-actions" data-module="track-button-click" data-track-category="form-button" data-track-action="taxonomy-tag-form-button">
-          <%= form.submit('Save topic changes', name: "save", class: "btn btn-lg btn-success") %>
-          <%= form.submit('Save and review legacy tagging', name: "legacy_tags", class: "btn btn-lg btn-primary") %>
+          <%= form.submit('Save topic changes', name: "save", class: "btn btn-lg btn-success",
+              data: {
+                  module: 'track-selected-taxons',
+                  track_category: 'taxonSelection',
+                  track_label: request.path
+              }) %>
+          <%= form.submit('Save and review legacy tagging', name: "legacy_tags", class: "btn btn-lg btn-primary",
+              data: {
+                  module: 'track-selected-taxons',
+                  track_category: 'taxonSelection',
+                  track_label: request.path
+              }) %>
           <span class="or_cancel">
             or
-            <%= link_to('cancel', admin_edition_path(@edition)) %>
+            <%= link_to('cancel', edit_admin_edition_tags_path(@edition)) %>
```

Comment:
How come this is changing?

Reply:
This is wrong, RubyMine getting a bit over-enthusiastic I think.